src/main.cpp

```diff
@@ -310,11 +322,26 @@ int main(int argc, char * argv[])
     if (!(ErrorDialogHandler::instance()->checkError())) {
         qDebug() << "Displaying mixxx";
         mixxx->show();
+        splash.finish(mixxx);
 
         qDebug() << "Running Mixxx";
         result = a.exec();
     }
 
+    QMessageBox cleaningUpBox;
+    cleaningUpBox.setStandardButtons(0);
+    cleaningUpBox.setWindowTitle(QObject::tr("Exiting Mixxx"));
+    cleaningUpBox.setText(QObject::tr(
+            "Cleaning up the library. This may take some time.\n"
+            "Do not restart Mixxx or shutdown the device until\n "
+            "this box has disappeared."));
```

Comment:
Since I use the eeePC it is one of our target machines. We have always took this as a low end reference in the past and there is no reason to drop this practice. 

Please be more explicit with your arguments. It is a new info for me that you are entirely against this PR. Except your last post, you have not written it. 

What is the reason for you opinion against the start up and shut down splash screens? 
I can't see a drawback. Do you have one?